pysatNASA/instruments/methods/ace.py

```diff
@@ -34,7 +34,9 @@
         }
 
 
-def load(fnames, tag='', inst_id='', to_pandas=False):
+# TODO(https://github.com/pysat/pysat/issues/1020): **kwargs can be removed
+# when header level meta is tested in version 3.3.0+ the core pysat
```

Comment:
hmm, I was hoping that this could be removed if we were testing upstream.
I don't like that it accepts kwargs it doesn't use just so the tests will pass